### PR TITLE
ticket 9 complete

### PR DESCRIPTION
**Pull request by jondon1337**

### CODE REVIEW

**Comment by jondon1337** [comment]
completed

File: __tests__/app.test.js [hunk 1]
```diff
@@ -107,6 +106,58 @@ describe("api/topics", () => {
       })
     });
   });
+  describe.only('api/articles', () => {
+    test.only('should respond with an articles array of article objects with the author set as the username from the users table, sorted in descending order', () => {
+      return request(app)
+      .get("/api/articles?sort_by=created_at&order=desc")
```
Comment (dav3rid) [inline __tests__/app.test.js:112]: This is a perfectly valid test as these queries are accepted. However, this should also be the case by default. Don't remove this test, but you can add another where no queries are applied and the sort order is automatically in place

File: models/model.js [hunk 2]
```diff
@@ -23,28 +23,54 @@ exports.fetchArticleById = (article_id) => {
...
 
+  if(!greenList.includes(sort_by) ) {
+    console.log("bad sortby")
+    return Promise.reject({ status: 400, msg: "Bad request"})
+  }
+  
+  if(order !== "DESC" && order !== "ASC") {
+    console.log("bad order")
+    return Promise.reject({ status: 400, msg: "Bad request"})
+  }
```
Comment (dav3rid) [inline models/model.js:62]: You could group these checks together as the resulting behaviour is the same - responding with a 400 bas request

File: models/model.js [hunk 3]
```diff
@@ -23,28 +23,54 @@ exports.fetchArticleById = (article_id) => {
...
+ 
+ 
+  let queryCreationTime =    `
```
Comment (dav3rid) [inline models/model.js:65]: haha I like this variable name - but I would suggest a change!

**Pull request merged**